GLOCKFILE

```diff
@@ -78,44 +75,44 @@ github.com/kr/pretty cfb55aafdaf3ec08f0db22699ab822c50091b1c4
 github.com/kr/text 7cafcd837844e784b526369c9bce262804aebc60
 github.com/leekchan/timeutil 28917288c48df3d2c1cfe468c273e0b2adda0aa5
 github.com/lib/pq d8eeeb8bae8896dd8e1b7e514ab0d396c4f12a1b
-github.com/lightstep/lightstep-tracer-go 91ae78ce073799b6a40e303d7dec6cc3a4257915
+github.com/lightstep/lightstep-tracer-go b1bc6c278e9a007584bbc0b9139e776b4e3203e3
```

Comment:
Did you intentionally pick up https://github.com/lightstep/lightstep-tracer-go/pull/58 here ?